docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -59,40 +59,19 @@ type: docs
 
 </ul>
 
-<br>The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+Before configuring the environment, consult [Prerequisites](../../prerequisites). 
```

Comment:
this link should point to the k8s section - https://deploy-preview-15391--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/#kubernetes-based-installations-1

Reply:
Changed link